Prüfschritte/de/11.1.3.1e Beschriftung von Formularelementen programmatisch ermittelbar.adoc

```diff
@@ -4,109 +4,65 @@ include::include/attributes.adoc[]
-
-In der Praxis kommt es immer wieder vor, dass Label und dessen Bedienelement
-getrennt voneinander vorgelesen werden.
-Dies ist meist dann der Fall, wenn Bedienelement und Label keine
-programmatische Verbindung haben.
-Um die Verbindung programmatisch herzustellen bieten die Plattformen
-unterschiedliche Mögglichkeiten an.
+. Zu Formularelementen per Wischgesten navigieren bzw. diese direkt antippen und prüfen, ob dessen Beschriftungen bzw. gegebnenfalls Gruppenbeschriftungen bei gruppierten Elementen korrekt ausgegeben werden.
+
+=== 3. Hinweise
+
+==== 3.1 Kombinierte Eingabeelemente
+
+Bei kombinierten Eingabeelementen hat nicht immer jedes Element eine zugeordnete Beschriftung. In diesem Fall sollen Elemente mit einer passenden (nicht sichtbaren) Bezeichnung für Screenreader versehen werden.
+
+Beispiel: In einem Formular werden für die Eingabe eines Datums drei Auswahllisten angeboten (Tag, Monat und Jahr). Die drei Auswahllisten haben eine gemeinsame Beschriftung: Datum. Die Auswahllisten für Tag, Monat und Jahr sind mit einer erklärenden Bezeichnung für Screenreader versehen. Diese Bezeichnungen sind dann nur von der Hilfstechnologie verfügbar, die entsprechenden Bezeichnungen sind nicht sichtbar.
+
+Wenn ein einfaches Suchformular nur aus einem Eingabefeld und einem Button besteht, ist oftmals keine sichtbare Beschriftung notwendig. Hier ist es ausreichend, wenn Eingabefeld und Button direkt nebeneinander positioniert sind, das Eingabefeld eine sinnvolle Textvorbelegung hat oder die Beschriftung des Buttons die Funktion eindeutig kennzeichnet (etwa "Suchen"). Das unbeschriftete Eingabefeld mit Textvorbelegung muss in solchen Fällen eine aussagekräftige (nicht sichtbare) Bezeichnung für den Screenreader haben, da für Screenreader-Nutzende der nachfolgende Button nicht gleichermaßen als Beschriftung taugt.
+
+==== 3.2 Verhindern des separaten Vorlesens von Label und Bedienelement
+
+In der Praxis kommt es immer wieder vor, dass Label und dessen Bedienelement getrennt voneinander vorgelesen werden. Dies ist meist dann der Fall, wenn Bedienelement und Label keine programmatische Verbindung haben. Um die Verbindung programmatisch herzustellen bieten die Plattformen unterschiedliche Mögglichkeiten an.
 In Apples iOS / iPadOS können diese Elemente
 https://developer.apple.com/documentation/uikit/accessibility/delivering_an_exceptional_accessibility_experience#3542986[
 gruppiert werden], sodass sie für den Screenreader als eine Einheit agieren.
 Für Android gibt es dazu https://developer.android.com/guide/topics/ui/accessibility/principles[
 vergleichbare Ansätze].
 
-=== Bewertung
+=== 3. Bewertung
 
-*Nicht erfüllt:*
+==== Nicht erfüllt:
 
-* Beschriftungen sind nicht korrekt mit den dazugehörigen Eingabefeldern
-  verknüpft.
+* Beschriftungen sind nicht korrekt mit den dazugehörigen Eingabefeldern verknüpft. Es wird bei Fokussierung des Feldes auch kein Name ausgegeben, der das Eingabefeld hinreichend beschreibt.
```

Comment:
Hier könnte man ewas ergänzen zu für sich genommen nicht ausreichend beschrifteten Bedienelementen wie ja oder nein, für die weder eine Gruppenbeschriftung noch ein vorangehender programmatisch ermittelbarer Text vorhanden ist (z.B. weil die Beschriftung nur in einem Text layer steckt, der vom Screenreader nicht ausgegeben wird). Klären mit App-Entwicklern...